src/utils/crypto.js

```diff
@@ -87,6 +87,19 @@ function verify (str, signature, publicKey) {
   return nacl.sign.detached.verify(new Uint8Array(str), signature, publicKey)
 }
 
+const personalMessageToBinary = (message) => {
```

Comment:
Please use `function` instead of assigning anonymous functions to `const`s unless you have a strong reason not to, e.g. because you're assigning a function returned by another function of higher order.